rest-api-spec/src/main/resources/rest-api-spec/api/engines.put.json

```diff
@@ -0,0 +1,39 @@
+{
+  "engines.put": {
```

Comment:
let's call it engine singular to match the URI?

Reply:
Done!